### DISCUSSION TITLE
Will AMD's be considered for use other than decoding?

### DISCUSSION THREAD

**Discussion by Dvalin21** [blakeblackshear/frigate#5588] [Ask A Question]
Will AMD GPU ever be considered for Tensorlfow like Nvidia? Thanks

**Comment by blakeblackshear** [comment]
Maybe someday with ROCm, but my understanding is that they are way far behind nvidia in this space.

https://docs.amd.com/bundle/Hardware_and_Software_Reference_Guide/page/Hardware_and_Software_Support.html

**Reply by Dvalin21** [reply]
Well crap...I guess neither here or there, but I was hoping not to have to buy another gpu and have one not being used. Other than Coral, what are some other alternatives I could try and utilize?

**Reply by NickM-27** [reply]
I mean you could use both GPUs at the same time without issue. I use my integrated AMD GPU and Nvidia GPU at the same time.

**Reply by Dvalin21** [reply]
I didn't even think of that. I will have to reconsider. Thanks!
